pg-auto-failover.spec

```diff
@@ -9,11 +9,11 @@ Summary:	Postgres extension for automated failover and high-availability
 Name:		%{sname}%{?pkginfix}_%{pgmajorversion}
 Provides:	%{sname}_%{pgmajorversion}
 Conflicts:	%{sname}_%{pgmajorversion}
-Version:	1.2
+Version:	1.3.citus
```

Comment:
Is this actually intended? Do we want this for consistency between packages for different extensions?

Reply:
Yes I guess the reason is that it's clear where the package comes from, i.e. built by us not pgdg.

Reply:
Yes but it's confusing, is it the package for pg_auto_failover with Citus support?

Reply:
That's true, I removed it now